README.md

```diff
@@ -35,70 +35,83 @@ So, what do you get by using this project as a template for your project? Let's
 ### For development on your computer
 
 1. Clone the repository to your computer and go to the `django-docker-template` directory:
-```console
-git clone https://github.com/amerkurev/django-docker-template.git
-cd django-docker-template
-```
+    ```console
```

Comment:
That's very odd, I've never seen PyCharm behave like this with README.md files. Plus, other projects don't have any indents in their README.md files. Please, let's not add any indents in this case.